financial-services-accelerator/components/org.wso2.financial.services.accelerator.event.notifications.service/pom.xml

```diff
@@ -204,8 +204,9 @@
                             org.wso2.carbon.identity.oauth2.*;version="${identity.inbound.auth.oauth.version.range}",
                             org.wso2.financial.services.accelerator.common.*;version="${project.version}",
                             org.wso2.financial.services.accelerator.consent.mgt.dao.*;version="${project.version}",
-                            org.wso2.financial.services.accelerator.consent.mgt.service.*;version="${project.version}",
+                            org.wso2.financial.services.accelerator.consent.mgt.service.*;version="${project.version}"
                         </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
As discussed offline, let's create an issue to remove this dynamic imports line in the future.

Reply:
Created issue https://github.com/wso2/financial-services-accelerator/issues/218